back/engines/commercial/bulk_import_ideas/app/controllers/bulk_import_ideas/web_api/v1/bulk_import_controller.rb

```diff
@@ -12,17 +12,18 @@ class WebApi::V1::BulkImportController < ApplicationController
           exporter_class: Exporters::IdeaXlsxFormExporter,
           parser_class: Parsers::IdeaXlsxFileParser
         },
-        'pdf' => {
-          exporter_class: Exporters::IdeaPdfFormExporter,
-          parser_class: Parsers::IdeaPdfFileParser
+        'html' => {
+          exporter_class: Exporters::IdeaHtmlFormExporter,
+          parser_class: nil # Not implemented for importing
         },
-        'htmlpdf' => {
+        'pdf' => {
           exporter_class: Exporters::IdeaHtmlPdfFormExporter,
           parser_class: Parsers::IdeaHtmlPdfFileParser
         },
-        'html' => {
-          exporter_class: Exporters::IdeaHtmlFormExporter,
-          parser_class: nil # Not implemented for importing
+        # The following classes are now for legacy support of the prawn based pdf import/export
```

Comment:
Why do we keep both pdf and legacy_pdf?

Reply:
Because when we import, we need to first load the form that the imported PDF was exported from. If customers already have surveys in the field then we will likely have problems because the forms are different. Probably need to leave the code in there for a few months.